<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryManager</class>
 <widget class="QWidget" name="HistoryManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="deleteB">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearAll">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>84</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear All History</string>
   </property>
  </widget>
  <widget class="HistoryTreeView" name="historyTree">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>711</width>
     <height>291</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HistoryTreeView</class>
   <extends>QTreeView</extends>
   <header>historytreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
